cli/run.go

```diff
@@ -27,9 +27,30 @@ const (
 	depManagerTryCheck = "_"
 )
 
-// --------------------
-// Utility
-// --------------------
+var runCommand = cli.Command{
+	Name:    "run",
+	Aliases: []string{"r"},
+	Usage:   "Runs a specified Workflow.",
+	Action:  run,
+	Flags: []cli.Flag{
+		// cli params
+		cli.StringFlag{Name: WorkflowKey, Usage: "workflow id to run."},
+		cli.StringFlag{Name: ConfigKey + ", " + configShortKey, Usage: "Path where the workflow config file is located."},
+		cli.StringFlag{Name: InventoryKey + ", " + inventoryShortKey, Usage: "Path of the inventory file."},
+		cli.BoolFlag{Name: "filtering", Usage: "Enables secret filtering", EnvVar: "BITRISE_SECRET_FILTERING"},
```

Comment:
If I see only the command without the help page, it might be not clearly understandable what `filtering` means.

I have no better suggestion like the ones below, but feel free ask someone else as well if you could find better, if not then avoid this comment.

if value true/false:
- `print-secrets`
- `hide-secrets`
- `filter-secrets`